bscript/opcodes.go

```diff
@@ -186,7 +186,9 @@ const (
 	OpCHECKMULTISIGVERIFY byte = 0xaf // 175
 	OpNOP1                byte = 0xb0 // 176
 	OpNOP2                byte = 0xb1 // 177
+	OpCHECKLOCKTIMEVERIFY byte = 0xb1 // 177
```

Comment:
these 2 fuckers are gone and good riddance to them haha

Reply:
Hahaha in the bin